utils/utils.go

```diff
@@ -86,11 +88,19 @@ func FinishNRTransaction(ctx context.Context, err error) {
 	}
 }
 
-//IgnoreNRTransaction ignores this NR trasaction and prevents it from being reported
+// IgnoreNRTransaction ignores this NR trasaction and prevents it from being reported
 func IgnoreNRTransaction(ctx context.Context) error {
 	t := GetNewRelicTransactionFromContext(ctx)
 	if t != nil {
 		return t.Ignore()
 	}
 	return nil
 }
+
+func ToJSONString(input interface{}) string {
+	b, err := json.Marshal(input)
+	if err != nil {
+		return ""
+	}
+	return *(*string)(unsafe.Pointer(&b))
```

Comment:
plain `string(b)` doesn't fit the bill here?
trying to understand the usecase of unsafe pointer. since we already know the input ([]byte) and output (string) types.

Reply:
this is for performance consideration - `string(b)` will allocate a new memory space and copy the data from byte[].
for this log every request/response purpose we can try to avoid it

https://stackoverflow.com/questions/59209493/how-to-use-unsafe-get-a-byte-slice-from-a-string-without-memory-copy